### PR TITLE
Feat technical debt 2 for database changes

### PR DESCRIPTION
**Pull request by fogelito**
This is for adjusting the technical dept PR changes on the database branch feat-technical-debt

### CODE REVIEW

File: tests/e2e/Scopes/Scope.php [hunk 1]
```diff
@@ -16,10 +18,9 @@ abstract class Scope extends TestCase
      */
     protected $client = null;
 
-    /**
-     * @var string
-     */
-    protected $endpoint = 'http://localhost/v1';
+    protected string $endpoint = 'http://localhost/v1';
+
+    public static Validator $dateValidator;
```
[the remark this answers is not in the document]
Comment (fogelito) [reply tests/e2e/Scopes/Scope.php]: Removed

File: tests/e2e/Scopes/Scope.php [hunk 2]
```diff
@@ -5,21 +5,16 @@
 use Appwrite\Tests\Retryable;
 use Tests\E2E\Client;
 use PHPUnit\Framework\TestCase;
-use Utopia\Database\ID;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Validator\DatetimeValidator;
```
Comment (abnegate) [inline tests/e2e/Scopes/Scope.php:9]: I think we can remove this now we're using instances

**Review by abnegate — COMMENTED** [review]
Looks good, just need to run the formatter the make the linter happy 👌

File: tests/e2e/Scopes/Scope.php (hunk 2 above)
Comment (fogelito) [reply tests/e2e/Scopes/Scope.php]: Yep.. removed some more use not in use :)

File: phpunit.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="false"
+         stopOnFailure="true"
```
[the remark this answers is not in the document]
Comment (fogelito) [reply phpunit.xml]: Done!

**Review by abnegate — APPROVED** [review]

File: composer.json [hunk 4]
```diff
@@ -49,7 +49,7 @@
         "utopia-php/cache": "0.8.*",
         "utopia-php/cli": "0.13.*",
         "utopia-php/config": "0.2.*",
-        "utopia-php/database": "0.28.*",
+        "utopia-php/database": "dev-feat-technical-debt as 0.28.0",
```
[the remark this answers is not in the document]
Comment (abnegate) [reply composer.json:52]: We can now update this to `0.29.*`

[the remark this answers is not in the document]
Comment (fogelito) [reply composer.json]: done forgot to update

**Review by lohanidamodar — APPROVED** [review]

**Pull request merged**